tests/setup_test.go

```diff
@@ -834,3 +834,11 @@ func directRun(t *testing.T, dir, config string, configFormat ConfigFormat, gate
 
 	t.Logf("directrun start out: %s", out)
 }
+
+type RefType int
+
+const (
+	RefTypeID RefType = iota
+	RefTypePath
+	RefTypeName
+)
```

Comment:
The consts should go with the other consts at the start of the file, not at the end.